sdk/openai/Azure.AI.OpenAI/src/Custom/AzureCognitiveSearchChatExtensionConfiguration.Serialization.cs

```diff
@@ -57,11 +58,25 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             }
             if (Optional.IsDefined(EmbeddingKey))
             {
-                writer.WriteString("embeddingKey"u8, EmbeddingKey);
+                writer.WritePropertyName("embeddingKey"u8);
+                writer.WriteStringValue(EmbeddingKey);
             }
-            // CUSTOM CODE NOTE: end of induced 'parameters' first, then the parent object
+
+            // CUSTOM CODE NOTE: End of induced 'parameters' first, then the parent object
             writer.WriteEndObject();
+
             writer.WriteEndObject();
         }
+
+        // CUSTOM CODE NOTE: This hides the AzureChatExtensionConfiguration.ToRequestContent method
+        //                      with the "override" keyword.
+
+        /// <summary> Convert into a Utf8JsonRequestContent. </summary>
```

Comment:
Try use `[CodeGenSuppress("ToRequestContent")]` on the class, as https://github.com/Azure/azure-sdk-for-net/pull/39460/files#diff-482ab30d94141b37bade0a7a96bf4973d713796e3b2580524160d97a996b4611

Reply:
I think the right behavior to override the behavior of the base class.